_episodes_rmd/07-vector-shapefile-attributes-in-r.Rmd

```diff
@@ -383,7 +383,7 @@ ggplot() +
   coord_sf()
 ```
 
-```{r plot-different-colors}
+```{r plot-different-colors fig.cap="Map of the paths in the study area using a different color palette."}
```

Comment:
```suggestion
```{r plot-different-colors, fig.cap="Map of the paths in the study area using a different color palette."}
```